pkg/tagger/global.go

```diff
@@ -61,10 +67,36 @@ func Init() {
 	})
 }
 
+// GetEntity returns the hash for the provided entity id.
+func GetEntity(entityID string) (*types.Entity, error) {
+	mux.RLock()
+	if captureTagger != nil {
+		entity, err := captureTagger.GetEntity(entityID)
+		if err == nil && entity != nil {
+			mux.RUnlock()
+			return entity, nil
+		}
+	}
+	mux.RUnlock()
+
+	return defaultTagger.GetEntity(entityID)
+}
+
 // Tag queries the defaultTagger to get entity tags from cache or sources.
```

Comment:
nit suggestion: `Tag queries the captureTagger (for replay scenarios) or defaultTagger...`